### DISCUSSION TITLE
Monetary contributions

### DISCUSSION THREAD

**Discussion by ztang28** [Cimbali/pympress#216] [Q&A] [answered]
Hi,

I found Pympress very helpful for me to present my Latex generated PDF slides in presenter mode (i.e. secondary screen, laser pointer mirroring, notes display). It is the one of the only software that works on Windows and doesn't require a ton of work to install. 

I don't know enough code to contribute to the project, but I'm willing to make monetary contributions. I think GitHub now allows your project to have a sponsor button. Would the project be open to it?

Thanks

Zhe

**Comment by Cimbali** [comment, marked as answer]
Hi Zhe! Thanks for your message, it’s nice too know pympress is appreciated. Money isn’t really part of the philosophy here, just enjoy the software!

There’s a number of options to contribute without coding if you feel like it:
- participate in [translating pympress](https://poeditor.com/join/project?hash=nKfRxeN8pS)
- try the [v1.7 beta](https://github.com/Cimbali/pympress/releases/tag/v1.7.0b1) (or generally the latest beta, if there is one) and report any bugs
- recommend pympress to anyone needing it
- some contributors also participate by packaging pympress on various platforms. This is more of a long-term engagement
